packages/libraries/router/src/registry_logger.rs

```diff
@@ -0,0 +1,98 @@
+use std::{env, fmt};
+
+static LOG_LEVEL_NAMES: [&str; 5] = ["ERROR", "WARN", "INFO", "DEBUG", "TRACE"];
+
+#[repr(usize)]
```

Comment:
I don't think this attribute is needed
```suggestion
```

Reply:
I think it's needed for a `>=` comparison, no?